it/controllers/CalculationControllerISpec.scala

```diff
@@ -615,6 +615,7 @@ class CalculationControllerISpec extends IntegrationSpecCommonBase with FeatureS
       parsedBody.select("#main-content .govuk-summary-list__row").get(1).select("dd").text() shouldBe "£0.00"
       parsedBody.select("#main-content .govuk-summary-list__row").get(2).select("dt").text() shouldBe "Left to pay"
       parsedBody.select("#main-content .govuk-summary-list__row").get(2).select("dd").text() shouldBe "£123.45"
+      parsedBody.select("#ttp-inset-text").isEmpty shouldBe true
```

Comment:
If you add the class I mentioned, you can then do 
```suggestion
      parsedBody.select(".whatever-class").isEmpty shouldBe true
```